src/teams.json

```
@@ -254,6 +254,28 @@
       "repos": [
         "GoogleCloudPlatform/workflows-samples"
       ]
+    },
+    {
+      "name": "ga-devrel",
+      "apis": [
+        "analytics-data"
+      ],
+      "repos": [
+        "googleapis/nodejs-analytics-data",
+        "googleapis/python-analytics-data",
+        "googleapis/java-analytics-data"
+      ]
+    },
+    {
+      "name": "ga-devrel",
+      "apis": [
+        "analytics-admin"
+      ],
+      "repos": [
+        "googleapis/nodejs-analytics-admin",
+        "googleapis/python-analytics-admin",
+        "googleapis/java-analytics-admin"
+      ]
     }
```

I'd like to use fewer lines to increase readability. We just need to make sure that the repos are correctly associated with the apis. WDYT?
```suggestion
    {
      "name": "ga-devrel",
      "apis": [
        "analyticsadmin"
        "analyticsdata"
      ]
    }
```